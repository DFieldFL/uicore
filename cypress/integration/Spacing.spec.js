/// <reference types="Cypress" />

context('Spacing: Margin and Padding', () => {
  before(() => {
    cy.visit('/spacing')
    cy.hideStickyHeader()
  })

  after(() => {
    cy.showStickyHeader()
  })

  it('snapshots for Spacing props', () => {
    cy.get('#spacing-margin-padding').toMatchSnapshot()
    cy.get('#spacing-margin-padding').toMatchImageSnapshot()
  })
})
