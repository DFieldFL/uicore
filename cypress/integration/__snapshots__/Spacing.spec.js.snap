exports[`Spacing: Margin and Padding > snapshots for Spacing props #0`] = `
<div
  id="spacing-margin-padding"
  class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---background-grey100--2FyT StyledProps---padding-huge--yMnr StyledProps---border--uz_- StyledProps---border-color-grey200--1Wrx"
>
  <div
    class="StyledProps---font--2ipy StyledProps---main--1Wws StyledProps---background-yellow100--1WvT StyledProps---margin-top-xsmall--33hS StyledProps---margin-right-small--Uojv StyledProps---margin-bottom-medium--1zkE StyledProps---margin-left-huge--Xd6r StyledProps---padding-top-small--1-XA StyledProps---padding-right-small--2ldZ StyledProps---padding-bottom-small--2Hba StyledProps---padding-left-large--28UT StyledProps---flex--2Jz8 StyledProps---border--uz_- StyledProps---border-color-yellow500--y3FU"
  >
    <p class="StyledProps---font--2ipy StyledProps---main--1Wws">
      The quick brown fox jumps over the lazy dog
    </p>
    <button
      type="button"
      class="bp3-button bp3-minimal StyledProps---font--2ipy Button---button--4ZD5 StyledProps---main--1Wws"
    >
      <span
        icon="cross"
        class="bp3-icon bp3-icon-cross StyledProps---main--1Wws"
        ><svg data-icon="cross" width="16" height="16" viewBox="0 0 16 16">
          <desc>cross</desc>
          <path
            d="M9.41 8l3.29-3.29c.19-.18.3-.43.3-.71a1.003 1.003 0 00-1.71-.71L8 6.59l-3.29-3.3a1.003 1.003 0 00-1.42 1.42L6.59 8 3.3 11.29c-.19.18-.3.43-.3.71a1.003 1.003 0 001.71.71L8 9.41l3.29 3.29c.18.19.43.3.71.3a1.003 1.003 0 00.71-1.71L9.41 8z"
            fill-rule="evenodd"
          ></path></svg
      ></span>
    </button>
  </div>
</div>
`;
